<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>444</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>160</horstretch>
    <verstretch>120</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>testCamIP_AXIS_213_PTZ</string>
  </property>
  <widget class="QPushButton" name="pbUp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>358</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVideo">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>labelVideo</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>321</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-170</number>
   </property>
   <property name="maximum">
    <number>170</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbZoomIn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom IN</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>26</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-10</number>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelPanValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>43</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PanValue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPlay">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStop">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCapture">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTiltValue">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>39</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TiltValue</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>101</width>
     <height>84</height>
    </rect>
   </property>
   <property name="title">
    <string>PTZ :</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelPan">
      <property name="text">
       <string>PAN :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelTilt">
      <property name="text">
       <string>TILT :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelZoom">
      <property name="text">
       <string>ZOOM :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelValeurZoom">
      <property name="text">
       <string>?</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelValeurTilt">
      <property name="text">
       <string>?</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelValeurPan">
      <property name="text">
       <string>?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbInfos">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Infos</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCapture">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>160</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>120</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>labelCapture</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDown">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>398</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DOWN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLeft">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LEFT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRight">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RIGHT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbZoomOut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>244</width>
     <height>139</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
